hc/settings.py

```diff
@@ -111,7 +112,7 @@
         }
     }
 
-# Update database configuration with $DATABASE_URL.
+
```

Comment:
The comment is necessary, in my view. Maybe it just needed editing but not removing it.